webapp/channels/src/components/threading/global_threads/global_threads.scss

```diff
@@ -110,8 +110,6 @@
     }
 
     @media screen and (max-width: 768px) {
-        grid-template-rows: 0 1fr;
```

Comment:
Not really sure what this was needed for anymore or why I left it in there when I did the overall layout/UI changes.